NLog.config

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <!-- 
+  See http://nlog-project.org/wiki/Configuration_file 
+  for information on customizing logging rules and outputs.
+   -->
+    <extensions>
+      <add assembly="NLog.Target.Datadog" />
+    </extensions>
+    <variable name="logDirectory" value="${basedir}/logs"/>
+    <variable name="source" value="Url-Shortener"/>
```

Comment:
Is there a version we could pull from somewhere to add a version tag?

Reply:
Nothing currently in the project. We could define one however

Reply:
@kthompson23 I have added an `AssemblyVersion` to the project which is injected into the logs and datadog tags. I'll add this to the Url Shortener PR as well